src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -8,36 +8,32 @@ namespace Stratis.Bitcoin.Configuration.Settings
     /// </summary>
     public class ConnectionManagerSettings
     {
+        /// <summary>Number of seconds to keep misbehaving peers from reconnecting (Default 24-hour ban).</summary>
+        public const int DefaultMisbehavingBantime = 24 * 60 * 60; 
+
         /// <summary>
         /// Default constructor.
         /// </summary>
         public ConnectionManagerSettings()
         {
+            this.Connect = new List<IPEndPoint>();
+            this.AddNode = new List<IPEndPoint>();
+            this.Listen = new List<NodeServerEndpoint>();
         }
 
         /// <summary>List of exclusive end points that the node should be connected to.</summary>
-        public List<IPEndPoint> Connect
-        {
-            get; set;
-        } = new List<IPEndPoint>();
+        public List<IPEndPoint> Connect { get; set; }
 
         /// <summary>List of end points that the node should try to connect to.</summary>
-        public List<IPEndPoint> AddNode
-        {
-            get; set;
-        } = new List<IPEndPoint>();
+        public List<IPEndPoint> AddNode { get; set; } 
 
         /// <summary>List of network interfaces on which the node should listen on.</summary>
-        public List<NodeServerEndpoint> Listen
-        {
-            get; set;
-        } = new List<NodeServerEndpoint>();
+        public List<NodeServerEndpoint> Listen { get; set; } 
 
         /// <summary>External (or public) IP address of the node.</summary>
-        public IPEndPoint ExternalEndpoint
-        {
-            get;
-            internal set;
-        }
+        public IPEndPoint ExternalEndpoint { get; internal set; }
+
+        /// <summary>Number of seconds to keep misbehaving peers from reconnecting.</summary>
+        public int BanTime { get; internal set; }
```

Comment:
i tend to have all these time values to reflect their units in the suffix - BanTimeSeconds, DefaultMisbehavingBantimeSeconds etc. what do you think?

Reply:
Yes good idea